core/src/main/java/org/frankframework/pipes/GetPrincipalPipe.java

```diff
@@ -33,13 +38,29 @@
  */
 @ElementType(ElementTypes.SESSION)
 public class GetPrincipalPipe extends FixedForwardPipe {
+	private @Getter String notFoundForwardName;
+	protected PipeForward notFoundForward;
+
+	@Override
+	public void configure() throws ConfigurationException {
+		super.configure();
+		if (StringUtils.isNotEmpty(getNotFoundForwardName())) {
```

Comment:
I often wonder wether it makes more sense to check `isEmpty()` or `isBlank()`.
What do you think is the better check in this case?

Reply:
As long as I am able to register a forward with a blank name, for example `<Forward name=" " path="doSomething" />`,  I should be able to specify the same value for the `notFoundForwardName` attribute.